docs/release-info/release-notes.md

```diff
@@ -57,50 +57,689 @@ For tips about how to write a good release note, see [Release notes](https://git
 
 This section contains important information about new and existing features.
 
+### Improved native queries
+
+Native queries can now group on nested columns and arrays.
```

Comment:
```suggestion
Native queries can now group on supported complex columns and nested arrays.
```

Reply:
Is it possible to expand upon the feature a bit more - maybe we can add an example and an image.

Reply:
We should mention that rollup on complex columns is supported via MSQ, and the possible advantages of doing so. I think we can combine it in the same section.